cypress/unit/store-mutation-createColumnToCategoryMap.cy.js

```diff
@@ -0,0 +1,38 @@
+import { getters, mutations } from "~/store/index-refactor";
+
+var store = {};
+
+describe("createColumnToCategoryMap", () => {
+
+    beforeEach(() => {
+
+        store = {
+
+            columnToCategoryMapping: {},
+
+            dataTable: [
+
+                {
+                    "column1Name": "value1", "column2Name": "value2"
+                },
+                {
+                    "column1Name": "value3", "column2Name": "value4"
+                }
+            ],
+
+            getColumnNames: p_state => ["column1Name", "column2Name"]
```

Comment:
> There are a few new complexities with the introduction of this mutation

Uff, ok, I tink I see what you mean. First: good catch, this is some tricky stuff. There are two things about this: 
1. Mutations get the `state`, not the `store`. So you can't access a `getter` from a mutation. The only reason this works here is because you are mocking it as a `state` method which isn't allowed. I commented more on the store side about how we might be able to address this.
2. So far we only made some simple `getters` that relied on the `state` and nothing else. This is probably OK for testing `getters` and `mutations` (because they only get the state anyway), but if we go towards `actions`, we have to be careful that we don't end up reimplementing half the store in every mock just to make things work. I'm mostly mentioning this here in case we decide to reimplement the mutation you created as an action (which I don't think we should do). In that case, we should probably make another issue and see if we can access the store directly in these tests (without having to mock all of the mutations and getters again).

So, long story short: this getter should go.